gui/gui-manager.cpp

```diff
@@ -647,6 +661,27 @@ void GuiManager::setLastMousePos(int16 x, int16 y) {
 	_lastMousePosition.time = _system->getMillis(true);
 }
 
+void GuiManager::setLanguageRTL() {
+	if (ConfMan.hasKey("guiRTL")) {
```

Comment:
Please add a comment here, something like "put guiRTL=yes to your scummvm.ini to enforce RTL"